src/components/Filters/SeverityContainer.tsx

```diff
@@ -1,8 +1,29 @@
 import React, { Component } from 'react';
 import Severity from '../Filters/Severity';
+import { Scenario, ScenarioList, SeverityLevelList, Stat } from "../../utils/constantsTypes";
 
-class SeverityContainer extends Component {
-    constructor(props) {
+interface Child {
+    key: string,
+    scenario: Scenario,
+    severity: Array<any>, //FIXME how to type a component
```

Comment:
@BenjiLee this doesn't look to be causing any issues so I'll go ahead and merge, but this is a TODO, right?

Reply:
Any just means that it can be any type which is pretty much no type checking but it allows us to add files without compilation errors if we run into something we don't want to resolve right away. It can become a bad habit but it's also nice that we don't need to know everything about typescript to get the benefits.